stock_picking_delivery_label_link/models/stock_picking.py

```diff
@@ -0,0 +1,36 @@
+# Copyright 2024 Tecnativa - David Vidal
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+from odoo import fields, models
+
+
+class StockPicking(models.Model):
+    _inherit = "stock.picking"
+
+    allowed_shipping_attachement_ids = fields.Many2many(
+        comodel_name="ir.attachment",
+        compute="_compute_allowed_shipping_attachement_ids",
+    )
+    shipping_label_ids = fields.Many2many(
```

Comment:
why did you choose a many2many and not a one2many ?

What is your use case when the same attachment is attached on different pickings ?